### PR TITLE
WZTo2L2Q cards update for UL

### PR DESCRIPTION
**Pull request by singh-ramanpreet**
Dear GEN,

Can you please review the cards for WZTo2L2Q ~and ZZTo2L2Q~ for UL production?

Thanks.

### CODE REVIEW

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/WZTo2L2Q01j_5f_NLO_FXFX/WZTo2L2Q01j_5f_NLO_FXFX_madspin_card.dat [hunk 1]
```diff
@@ -1,13 +1,9 @@
 set ms_dir ./madspingrid
-
 set Nevents_for_max_weigth 250 
 set max_weight_ps_point 400  
 set BW_cut 40
```
Comment (sihyunjeon) [inline bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/WZTo2L2Q01j_5f_NLO_FXFX/WZTo2L2Q01j_5f_NLO_FXFX_madspin_card.dat:4]: Can you confirm with MadGraph authors that giving 2 different BW cutoffs in MG (bwcutoff=15) and MS (bwcutoff=40) are fine?
I remember him saying it's not physical

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/WZTo2L2Q01j_5f_NLO_FXFX/WZTo2L2Q01j_5f_NLO_FXFX_run_card.dat [hunk 2]
```diff
@@ -93,6 +97,8 @@ $DEFAULT_PDF_SETS = lhaid     ! if pdlabel=lhapdf, this is the lhapdf number
   0.5     = rw_Fscale_down   ! lower bound for fact scale variations
   2.0     = rw_Fscale_up     ! upper bound for fact scale variations
 $DEFAULT_PDF_MEMBERS = reweight_PDF     ! reweight to get PDF uncertainty
+      ! First of the error PDF sets
+      ! Last of the error PDF sets
```
Comment (sihyunjeon) [inline bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/WZTo2L2Q01j_5f_NLO_FXFX/WZTo2L2Q01j_5f_NLO_FXFX_run_card.dat:101]: remove from reweight_PDF to here.
You double defined reweight_PDF 

and below 2 shouldn't exist (although it's commented  out)
       ! First of the error PDF sets
       ! Last of the error PDF sets

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/ZZTo2L2Q01j_5f_NLO_FXFX/ZZTo2L2Q01j_5f_NLO_FXFX_proc_card.dat [hunk 3]
```diff
@@ -1,12 +1,9 @@
 import model loop_sm-ckm_no_b_mass
 define p = p b b~
 define j = j b b~
-
 define V = z
 define ell+ = e+ mu+ ta+
 define ell- = e- mu- ta-
-
 generate p p > V ell+ ell- [QCD] @0
 add process p p > V ell+ ell- j [QCD] @1
```
Comment (sihyunjeon) [inline bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/ZZTo2L2Q01j_5f_NLO_FXFX/ZZTo2L2Q01j_5f_NLO_FXFX_proc_card.dat:8]: This is also similar to the last comment I made in [1].
I don't think this is safe to use

[1] https://github.com/SMP-GEN-Coordination/mc-requests/issues/2

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/ZZTo2L2Q01j_5f_NLO_FXFX/ZZTo2L2Q01j_5f_NLO_FXFX_run_card.dat [hunk 4]
```diff
@@ -93,8 +96,9 @@ average = event_norm ! Normalize events to sum or average to the X sect.
   2.0     = rw_Rscale_up     ! upper bound for ren scale variations
   0.5     = rw_Fscale_down   ! lower bound for fact scale variations
   2.0     = rw_Fscale_up     ! upper bound for fact scale variations
- 292201   = PDF_set_min      ! First of the error PDF sets 
- 292302   = PDF_set_max      ! Last of the error PDF sets
+$DEFAULT_PDF_MEMBERS = reweight_PDF     ! reweight to get PDF uncertainty
+      ! First of the error PDF sets
+      ! Last of the error PDF sets
```
Comment (sihyunjeon) [inline bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/ZZTo2L2Q01j_5f_NLO_FXFX/ZZTo2L2Q01j_5f_NLO_FXFX_run_card.dat:101]: same as above

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/ZZTo2L2Q01j_5f_NLO_FXFX/ZZTo2L2Q01j_5f_NLO_FXFX_proc_card.dat (hunk 3 above)
Comment (singh-ramanpreet) [reply bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/ZZTo2L2Q01j_5f_NLO_FXFX/ZZTo2L2Q01j_5f_NLO_FXFX_proc_card.dat:8]: @sihyunjeon
Okay. I will put ZZ one on a hold for now.

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/WZTo2L2Q01j_5f_NLO_FXFX/WZTo2L2Q01j_5f_NLO_FXFX_madspin_card.dat (hunk 1 above)
Comment (singh-ramanpreet) [reply bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/WZTo2L2Q01j_5f_NLO_FXFX/WZTo2L2Q01j_5f_NLO_FXFX_madspin_card.dat:4]: @sihyunjeon 
I am not very experienced with MG/MS. I will take your suggestion as it is. If it's not physical, I can match them either bwcutoff=15 or 40.

Comment (sihyunjeon) [reply bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/WZTo2L2Q01j_5f_NLO_FXFX/WZTo2L2Q01j_5f_NLO_FXFX_madspin_card.dat:4]: Your m(ll from Z) will be (91 - width X bwcutoff) ~ (91 + width X bwcutoff). If you are fine with such changes, I'll just suggest to use bwcutoff=15 for both MG and MS as this is the MG default choice

**Comment by sihyunjeon** [comment]
Can you remove the ZZ cards from here?
And also change bwcutoff? I'll be happy to merge by then

**Comment by Saptaparna** [comment]
I agree that the BWcutoff should be set to 15 \sigma (one should operate in the NWA) and should be consistent between MG and MS.

**Comment by singh-ramanpreet** [comment]
Hi, I forced push to remove ZZ from the commits.
I also changed BW to 15 in both MG and MS.
Thanks!

**Comment by sihyunjeon** [comment]
Changed one line (the commented out lines) to make things more clear

**Pull request merged**